scripts/EN/Harry

```diff
@@ -7329,26 +7329,26 @@ Area(0x107); // 0x94b4: 00001026
 CleanScreen(); // 0x94ba: 00000064
 LoadBackground(0xab); // 0x94be: 00001069
 DrawBackground(); // 0x94c4: 00000065
-ShowText((3007)"After looking over the manuscript again I let out a long sign, and neatly pushed the stack of papers together and placed them in the envelope I had prepared."); // 0x94c8: 0000100c
+ShowText((3007)"After looking over the manuscript again I let out a long sigh, W1and neatly pushed the stack of papers together and placed them in the envelope I had prepared."); // 0x94c8: 0000100c
 HandleInput(); // 0x94ce: 00000010
-ShowText((3008)"I licked the seal of the envelope and then tightly sealed it. The act of properly sealing the envelope with such feeling seemed like a certain kind of ceremony."); // 0x94d2: 0000100c
+ShowText((3008)"I licked the seal and then tied a string around it to tightly seal the envelope.¥nW2The act of properly sealing the envelope with such feeling seemed like a certain kind of ceremony."); // 0x94d2: 0000100c
```

Comment:
Added the bit about tying the string around the envelope.

Missed this due to typing the wrong kanji.